connectors/sources/sharepoint_online.py

```diff
@@ -606,9 +606,14 @@ async def drive_items(self, drive_id, url=None):
             yield page
 
     async def drive_item_permissions(self, drive_id, item_id):
```

Comment:
So there's no way to fetch permissions in batch?

Do you think doing [request batching](https://learn.microsoft.com/en-us/graph/json-batching) could help here?

Reply:
Just for my understanding: We would pile up to 20 drive items and then fetch the permissions for these 20 at once?

Reply:
Yup. Code will get ugly, but that's potentially 20x performance bump?

Reply:
Let's do it, if it improves performance 🚀